app/src/main/java/org/oppia/android/app/player/state/StateFragment.kt

```diff
@@ -19,9 +20,15 @@ import org.oppia.android.app.player.state.listener.PreviousResponsesHeaderClickL
 import org.oppia.android.app.player.state.listener.ReturnToTopicNavigationButtonListener
 import org.oppia.android.app.player.state.listener.ShowHintAvailabilityListener
 import org.oppia.android.app.player.state.listener.SubmitNavigationButtonListener
+import org.oppia.android.util.extensions.getProto
 import org.oppia.android.util.extensions.getStringFromBundle
+import org.oppia.android.util.extensions.putProto
 import javax.inject.Inject
 
+private const val STATE_FRAGMENT_RAW_USER_ANSWER_KEY = "StateFragment.raw_user_answer"
+private const val STATE_FRAGMENT_HAS_PREVIOUS_HEADER_COLLAPSED =
```

Comment:
Rename to `STATE_FRAGMENT_IS_PREVIOUS_RESPONSES_HEADER_EXPANDED_KEY` = `"StateFragment.is_previous_responses_expanded_collapsed"`

Reply:
In onCreate you are creating a variable `hasPreviousResponsesExpanded ` and here you are using key `STATE_FRAGMENT_HAS_PREVIOUS_HEADER_COLLAPSED `

This is actually incorrect, either focus only on `collapsed` or focus only on `expanded` both are literally opposite terms so they should not be mixed to avoid confusion. 

Suggest using `expanded` only as we are using this in viewmodels/xml files too.

Reply:
Updated